lxd/instance/drivers/driver_lxc.go

```diff
@@ -6368,10 +6376,14 @@ func (d *lxc) IsPrivileged() bool {
 	return shared.IsTrue(d.expandedConfig["security.privileged"])
 }
 
+// isRunningStatusCode returns if instance is running from status code.
+func (d *lxc) isRunningStatusCode(statusCode api.StatusCode) bool {
+	return statusCode != api.Error && statusCode != api.Stopped
+}
+
 // IsRunning returns if instance is running.
 func (d *lxc) IsRunning() bool {
-	state := d.State()
-	return state != "BROKEN" && state != "STOPPED"
```

Comment:
@brauner @stgraber I've not been able to find a scenario where state can be "BROKEN", so I think this part is safe to remove. The `isRunningStatusCode` function above handles scenarios where `getLxcState` returns `api.Error` and treats it as not-running in that case, which I think is similar to what checking for "BROKEN" was intended for here.